features/networking/sctp.feature

```diff
@@ -26,48 +27,45 @@ Feature: SCTP related scenarios
       | ["spec"]["nodeName"]      | <%= cb.workers[1].name %> |
     Then the step should succeed
     And the pod named "sctpclient" becomes ready
-
+   
```

Comment:
This might be an unexpected change.

Reply:
@liangxia Sorry, not get your point here, could you explain more specifically?

Reply:
White spaces introduced on the line.

Reply:
Fixed, thanks!